### DISCUSSION TITLE
Supported browsers and devices detection

### DISCUSSION THREAD

**Discussion by mercem** [google/model-viewer#2809] [Q&A] [answered]
Hi,

Is there a way to understand if current browser and device is supported? In the documentation it says, `supported on the last 2 major versions of all evergreen desktop and mobile browsers.` but is there an API that we can use to detect it or should we manually check browser and device?

**Comment by elalish** [comment, marked as answer]
We do our best to test a fairly wide range of devices and browsers and fix any problems, but for 100% certainty,  yes you should  do your own testing  (and file bugs!).
